tasks/typechecking-stats.js

```diff
@@ -0,0 +1,67 @@
+import rawGlob from 'glob';
+import json5 from 'json5';
+import fs from 'node:fs/promises';
+import { promisify } from 'node:util';
+
+const globOne = promisify(rawGlob);
+
+/**
+ * @param {string[]} patterns
```

Comment:
nitpick: I prefer `Array<string>` instead of `string[]`. Do what you want with that.

Reply:
I have two wishes:

* that we (the JS world) did not have both similar options in the first place
* that we (the component maintainers) always did the same

I updated to `Array<string>` :wink: